### ISSUE TITLE
fpm-setup action broken by Continuous Deployment Changes

### ISSUE THREAD

**Issue by everythingfunctional** [fortran-lang/setup-fpm#6]
### Description

Not sure whether to report the bug here or at the fpm-setup repo, but the latest changes appear to have broken that action.

### Expected Behaviour

The action is trying to download the "latest" version of fpm, which is now, presumably the "current" Release, but the binaries don't follow the same naming convention (i.e. `fpm-<version>-linux-x86_64`. Is the recommended usage to specify an fpm version with the action, or should something be changed?

### Version of fpm

current

### Platform and Architecture

Linux, Windows and MacOS

### Additional Information

Output from running the action

```
Run fortran-lang/setup-fpm@v3
  with:
    github-token: ***
    use-haskell: false
    fpm-version: latest
    fpm-repository: https://github.com/fortran-lang/fpm
  env:
    FC: gfortran
    GCC_V: 11
use-haskell: false
fpm-version: latest
fpm-repository: https://github.com/fortran-lang/fpm
This platform is linux
Fetching fpm from https://github.com/fortran-lang/fpm/releases/download/current/fpm-current-linux-x86_64
Error: Error while trying to fetch fpm - please check that a version exists at the above release url.
undefined
Error: The "path" argument must be of type string. Received type undefined
```

**Comment by awvwgk** [comment]
When selecting the latest version, the latest stable version should be targeted rather than the one we deploy continuously.

**Comment by everythingfunctional** [comment]
Looking at the code that determines the latest release version ([here](https://github.com/fortran-lang/setup-fpm/blob/7235df17dca5324ce4fca2bc3a86f7b22a5baa01/index.js#L139)), it seems that we can now assume that `current` will always be the "latest", so we should take the second (e.g. index 1 instead of 0), instead of the first release [here](https://github.com/fortran-lang/setup-fpm/blob/7235df17dca5324ce4fca2bc3a86f7b22a5baa01/index.js#L147).

I can submit a PR if you think that's correct and you don't want to make the change yourself.

**Comment by awvwgk** [comment]
There is always the possibility to check for the latest release https://github.com/fortran-lang/fpm/releases/latest rather than checking for the newest tag.

**Comment by awvwgk** [comment]
Fixed by #7
